gulpfile.js

```diff
@@ -176,6 +193,11 @@ function lintTask() {
     .pipe(eslint.failAfterError());
 }
 
+function lintHtmlTask() {
+  return gulp.src('./samples/**/*.html')
+    .pipe(htmllint(htmllintOptions));
```

Comment:
No need to require `htmllintOptions`, I think [this](https://www.npmjs.com/package/gulp-htmllint#optionsconfig) should work:

```javascript
  .pipe(htmllint({
    config: '.htmllint.js' // or .yml is supported
  }));
```

Reply:
Looking at the source, gulp-htmllint only supports proper JSON
https://github.com/yvanavermaet/gulp-htmllint/blob/master/src/index.js#L20